sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4033,6 +4033,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val CLI_PRINT_CURRENT_DB =
+    buildConf("spark.sql.cli.print.currentDb")
```

Comment:
if this is a bug fix, then should this be a legacy config? `spark.sql.legacy.emptyCurrentDBInCli`.